app/controllers/concerns/generic_crud.rb

```diff
@@ -1,12 +1,6 @@
 module GenericCrud
```

Comment:
should not be a concern. it should be a base controller and diffrent controller should inherit it